cfgov/v1/wagtail_hooks.py

```diff
@@ -163,3 +168,84 @@ class MegaMenuModelAdmin(ModelAdmin):
 def clear_mega_menu_cache(sender, instance, **kwargs):
     from django.core.cache import caches
     caches['default_fragment_cache'].delete('mega_menu')
+
+
+def get_resource_tags():
+    tag_list = []
+
+    for resource in Resource.objects.all():
```

Comment:
I suspect this lookup could be done in a more optimal way. This probably doesn't matter much given the size of the data being queried, though.